tests/test_server_interface.py

```diff
@@ -12,3 +13,12 @@ def test_read_file_file_exists():
         actual = server.read_file("english_alphabets.txt")
         expected = "ABCDEFGHIJKLMNOPQRSTUVWXYZ".encode("UTF-8")
         assert actual == expected
+
+    @staticmethod
+    def test_delete_file():
+        server_file_directory = os.getcwd()
+        f = open(f"{server_file_directory}/deleteme.txt", "w")
```

Comment:
We might want to close the file with something like
```
with open(f"{server_file_directory}/deleteme.txt", "w") as f:
    # Perform file operations
    pass
```

Reply:
sure. Alternatively f.close() would do as well but either is fine.

Reply:
Sure, that works too